lib/listener.js

```diff
@@ -16,64 +16,65 @@ Listener.prototype.startGame = function () {
 };
 
 Listener.prototype.setListeners = function () {
-  this.firing(this.player, this.projectiles);
-  this.moveRight(this.player);
-  this.moveLeft(this.player);
-  this.toggleProjectileType(this.player);
+  this.firing();
+  this.moveRight();
+  this.moveLeft();
+  this.toggleProjectileType();
 };
```

Comment:
This is way cleaner now that you aren't passing in a bunch of parameters to each method.